types/api/v1beta1/event.proto

```diff
@@ -0,0 +1,162 @@
+syntax = "proto3";
+
+option go_package = "github.co/aquasecurity/tracee/api/v1beta1";
+
+package tracee.v1beta1;
+
+message Event {
+    int64 timestamp = 1;
+    int64 id = 2;
+    string name = 3;
+    // TODO: version? are we going to have version here, or just at the definition?
+    map<string, string> tags = 4; // TODO:  is this type correct?
+    map<string, string> labels = 5; // TODO:  is this type correct?
+    Policies policies = 6;
+    Context context = 7;
+    repeated EventData data = 8;
+    optional Threat threat = 9;
+}
+
+message Policies {
+    repeated string matched = 1;
+    repeated string actions = 2;
+}
+
+message Context {
+    Process process = 1;
```

Comment:
Theoretically we may have events that are not specific to some process (e.g. a threat detected since some processes communicated in some way) so maybe we should make it optional as well?